apps/api/src/database/schema/util.ts

```diff
@@ -0,0 +1,15 @@
+import { numeric, varchar } from "drizzle-orm/pg-core";
+
+/**
+ * PG column for amount of tokens in humanese
+ */
+export function tokenAmount(name: string) {
+  return numeric(name, { precision: 30, scale: 18, mode: "number" });
```

Comment:
sorry to keep commenting on things, but it's hard to see on the nuances of the schema all at once. I feel like the  `mode: "number"` is not enough to represent token amount correctly, and we should use bigint. but I don´t know enough about these conversions from postgres to typescript

Reply:
@mzkrasner / @joewagner i know you guys dealt with some number precision issues. is there a reason we chose the pattern in the snippet (or see [here](https://github.com/recallnet/js-recall/blob/main/apps/api/src/database/schema/trading/defs.ts#L157) in the `main` branch) vs bigints?